openmc/model/model.py

```diff
@@ -31,6 +40,14 @@ class Model:
         Tallies information
     plots : openmc.Plots, optional
         Plot information
+    chain_file : str or Path, optional
+        Path to the depletion chain XML file.  Defaults to the chain
+        found under the ``depletion_chain`` in the
+        :envvar:`OPENMC_CROSS_SECTIONS` environment variable if it exists. If a
+        str is provided it will be converted to a Path object.
+    fission_q : dict, optional
+        Dictionary of nuclides and their fission Q values [eV].
+        If not given, values will be pulled from the ``chain_file``.
```

Comment:
My decision to store this as an attribute is because I envision ``model.deplete`` being called multiple times in a given analysis. Since the operator kwargs are unlikely to change for each of these ``model.deplete`` calls, it seems as if that should be saved somewhere to make the user input burden smaller.

Having said that, I do recognize that the ``operator_kwargs`` is a good idea. 
How do you feel about replacing ``Model.chain_file`` and ``Model.fission_q`` with ``Model.operator_kwargs``?

Reply:
Aaalright fine. After trying it out, I decided I agree, operator_kwargs could change and thus no reason to set it on the Model class. So I did as you suggested (pass operator_kwargs and integrator_kwargs to deplete and removed the operator parameters from Model)